parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -197,28 +197,40 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
+    version = 2
     subreaper = false
     oom_score = 0
-    [plugins.cri]
-    sandbox_image = "{{GetPodInfraContainerSpec}}"
-    [plugins.cri.containerd.untrusted_workload_runtime]
-    runtime_type = "io.containerd.runtime.v1.linux"
-    {{if IsKataContainerRuntime }}
-    runtime_engine = "/usr/bin/kata-runtime"
-    {{else}}
-    runtime_engine = "/usr/local/sbin/runc"
-    {{end}}
-    [plugins.cri.containerd.default_runtime]
-    runtime_type = "io.containerd.runtime.v1.linux"
-    {{if IsKataContainerRuntime }}
-    runtime_engine = "/usr/bin/kata-runtime"
-    {{else}}
-    runtime_engine = "/usr/local/sbin/runc"
-    {{end}}
-    {{if IsKubenet }}
-    [plugins.cri.cni]
-    conf_template = "/etc/containerd/kubenet_template.conf"
 
+    [plugins]
```

Comment:
is the add'l indentation significant, or just there to aid maintenance?

Reply:
Yeah I don't think toml cares about indentation. I just found it easier to read.